app/src/main/java/chat/rocket/android/helper/OkHttpHelper.java

```diff
@@ -48,4 +63,28 @@ public static OkHttpClient getClientForWebSocket() {
     }
     return httpClientForWS;
   }
+
+  public static String getContentType(String uri) {
+    Request request = new Request.Builder()
+        .url(uri)
+        .head()
+        .build();
+    getDefaultHttpClient().newCall(request).enqueue(new Callback() {
+      @Override
+      public void onFailure(@NonNull Call call, @NonNull IOException ioException) {
+        ioException.printStackTrace();
+      }
+
+      @Override
+      public void onResponse(@NonNull Call call, @NonNull final Response response)
+          throws IOException {
+        if (!response.isSuccessful()) {
+          throw new IOException("Unexpected code: " + response);
+        } else {
+          contentType = response.header("Content-Type");
+        }
+      }
+    });
+    return contentType;
```

Comment:
We should not do this, if we really need to get the Content-Type, we will need to implement our own Image loading library, like Fresco or Picasso, to download, cache and display it...

Doing a lot of `HEAD`s here will make a lot of useless requests on fling, we are not caching the result...

Ideally this should be handled on the Image library, on Fresco, or switch to some that already supports SVG.